cwl/src/main/scala/cwl/CommandOutputBinding.scala

```diff
@@ -19,6 +19,24 @@ case class CommandOutputBinding(glob: Option[Glob] = None,
                                 outputEval: Option[StringOrExpression] = None)
 
 object CommandOutputBinding {
+  /**
+   * TODO: WOM: WOMFILE: Need to support returning globs for primary and secondary.
+   *
+   * Right now, when numerous glob files are created, the backends place each glob result into a different subdirectory.
+   * Later, when the secondary files are being generated each of their globs are in different sub-directories.
+   * Unfortunately, the secondary file resolution assumes that the secondaries are _next to_ the primaries, not in
+   * sibling directories.
+   *
+   * While this is being worked on, instead of looking up globs return regular files until this can be fixed.
+   */
+  def dependingIfGlobLike[A](path: String, isGlobLike: => A, isNotGlobLike: => A): A = {
```

Comment:
ToL IMO this should be `isGlobLike` and return a boolean and let user use `if`/`fold` semantics